encodings/fastlanes/src/bitpacking/mod.rs

```diff
@@ -112,19 +112,15 @@ impl BitPackedArray {
 
     #[inline]
     pub fn patches(&self) -> Option<Array> {
-        if self.metadata().patches_len > 0 {
-            Some(
-                self.array()
-                    .child(
-                        1,
-                        &self.dtype().with_nullability(Nullability::Nullable),
-                        self.metadata().patches_len,
-                    )
-                    .expect("Missing patches array"),
```

Comment:
I wouldn't expect this function to change

Reply:
yep, this was wrong, good catch